server/resources/resource_management.go

```diff
@@ -480,7 +480,7 @@ func GetSingleResource(context middleware.Context, dataStore db.DB, resourceSche
 //GetSingleResourceInTransaction get resource in single transaction
 func GetSingleResourceInTransaction(context middleware.Context, resourceSchema *schema.Schema, resourceID string, tenantIDs []string, domainIDs []string) (err error) {
 	defer MeasureRequestTime(time.Now(), "get.single.in_tx", resourceSchema.ID)
-	var options *transaction.ViewOptions
+	options := &transaction.ViewOptions{Details: true}
```

Comment:
why join is used for get single resource?

Reply:
To preserve backward compatibility for API. Please note that currently UI depends on that.

Reply:
but in that case it's rewritten 2 lines below

Reply:
> (not counting JS)

I'm afraid this sentence disqualifies the change as we still rely on JS extensions in tests. And there is no benefit in discarding compatibility with JS just to remove single `if` statement